app/controllers/application_controller.rb

```diff
@@ -1,2 +1,8 @@
 class ApplicationController < ActionController::Base
+    
+    private
+    
+    def load_categories
```

Comment:
Because we have then added the load_categories to the article controller as before action, so that every time a new article is created in the application before it happens we are actually loading the categories.